cypress/integration/projectSettings/project_settings.ts

```diff
@@ -83,4 +96,18 @@ describe("Repo Settings", () => {
   it("Does not show a 'Default to Repo' button on page", () => {
     cy.dataCy("default-to-repo").should("not.exist");
   });
+
+  it("Does not show a 'Move to New Repo' button on page", () => {
+    cy.dataCy("move-repo-button").should("not.exist");
+  });
+
+  it("Selects a default logger on click", () => {
+    cy.get("[data-testid=leafygreen-ui-select-menubutton]").click();
+    cy.get("[data-leafygreen-ui=option]").first().click();
```

Comment:
I'm a little reluctant to use leafygreens internal selectors to select elements in our tests. Since there is no guarantee that they will not change. We can alternatively select the elements based off of their contents.  
```suggestion
    cy.contains("Select Default Logger").click();
    cy.contains("evergreen").click();
```